pkg/docker/docker.go

```diff
@@ -100,22 +104,23 @@ type PullResult struct {
 
 // RunContainerOptions are options passed in to the RunContainer method
 type RunContainerOptions struct {
-	Image           string
-	PullImage       bool
-	PullAuth        docker.AuthConfiguration
-	ExternalScripts bool
-	ScriptsURL      string
-	Destination     string
-	Command         string
-	Env             []string
-	Stdin           io.Reader
-	Stdout          io.Writer
-	Stderr          io.Writer
-	OnStart         func() error
-	PostExec        PostExecutor
-	TargetImage     bool
-	NetworkMode     string
-	User            string
+	Image            string
+	PullImage        bool
+	PullAuth         docker.AuthConfiguration
+	ExternalScripts  bool
+	ScriptsURL       string
+	Destination      string
+	Command          string
+	CommandOverrides func(originalCmd string) string
+	Env              []string
+	Stdin            io.Reader
+	Stdout           io.Writer
+	Stderr           io.Writer
+	OnStart          func(string) error
```

Comment:
Did this change because of the bump on the docker client lib?!
What does it do?